api/nodes/serializers.py

```diff
@@ -110,6 +118,7 @@ def get_pointers_count(self, obj):
         return len(obj.nodes_pointer)
 
     def create(self, validated_data):
+        validated_data.pop('type')
```

Comment:
Perhaps we use custom fields for id and type:

```python
class NodeContributorsSerializer(JSONAPISerializer):
    """ Separate from UserSerializer due to necessity to override almost every field as read only
    """
    id = IDField(source='_id')
    type = TypeField()  # required and write-only by default
```

This solves a few problems:

- We can enforce that type is required and write-only (by default)--that way, devs don't have to remember to include those params.
- JSONAPISerializer could exclude these fields in validated data. You will need to take a look at the base DRF Serializer implementation to make this work.